Localize/lang/strings.json

```diff
@@ -526,6 +527,7 @@
   "TPV6j/": "Paging count invalid",
   "TRpSCQ": "Action Type",
   "TUaunO": "Enter a value",
+  "TVrS/G": "Whole view",
```

Comment:
Elliott suggested the name "Global View" and I think I like it better then our current choices